opengemini/write.go

```diff
@@ -87,7 +87,11 @@ func (c *client) WritePoint(ctx context.Context, database string, point *Point,
 
 	resp, err := c.innerWrite(database, &buffer)
 	if err != nil {
-		callback(errors.New("innerWrite request failed, error: " + err.Error()))
+		err := errors.New("innerWrite request failed, error: " + err.Error())
+		if callback != nil {
+			callback(err)
+		}
+		return err
```

Comment:
```suggestion
		return nil
```

I prefer to `return nil` here since we decide to handle this error in callback.
And it's clear that anyone who didn't pass callback will miss error, callback should be pass in production.